diff --git a/node_modules/monaco-languageclient/lib/console-window.js b/node_modules/monaco-languageclient/lib/console-window.js
index a52339f..d980e8e 100644
--- a/node_modules/monaco-languageclient/lib/console-window.js
+++ b/node_modules/monaco-languageclient/lib/console-window.js
@@ -17,12 +17,14 @@ class ConsoleWindow {
         if (type === services_1.MessageType.Warning) {
             console.warn(message);
         }
+        /*
         if (type === services_1.MessageType.Info) {
             console.info(message);
         }
         if (type === services_1.MessageType.Log) {
             console.log(message);
         }
+         */
         return Promise.resolve(undefined);
     }
     createOutputChannel(name) {
@@ -32,10 +34,10 @@ class ConsoleWindow {
         }
         const channel = {
             append(value) {
-                console.log(name + ': ' + value);
+                // console.log(name + ': ' + value);
             },
             appendLine(line) {
-                console.log(name + ': ' + line);
+                // console.log(name + ': ' + line);
             },
             show() {
                 // no-op
